protocols/migration/migration_reports_5_0_0_to_reports_6_0_0.py

```diff
@@ -25,3 +25,172 @@ def migrate_interpretation_request_rd(self, old_instance):
         return self.validate_object(
             object_to_validate=new_instance, object_type=self.new_model.InterpretationRequestRD
         )
+
```

Comment:
This code does a lot of copying a property with the same name between objects. I don't think there are any errors but it's easy to see how there could be. It would be quite difficult to work out if an optional property is missed for example.

Would it make sense to automatically copy all the properties that exist in both or something? This would also make it clearer what the changes actually are. It would also make it less explicit so it might be a bad idea.

Reply:
AFAIK that's what `convert_class` does

Reply:
`convert_class` will copy over things that are the same name in both, but only if they're a primitive type. If they're an object themselves then that's why the migrations need to be done. Like `migrate_variants` below, the field is still called `variants` but it needs to be a v6 `Variants` object not a v5 one.

Reply:
Oh ok. I was looking at line like this though 
```
         new_variant_attributes.genomicChanges = old_variant.genomicChanges
         new_variant_attributes.cdnaChanges = old_variant.cdnaChanges
         new_variant_attributes.proteinChanges = old_variant.proteinChanges
```
Does that mean these would be copied over by the previous covert_class?

Reply:
in principle, yes

Reply:
Either way, it's probably more complicated than I think it is and might not be worth fixing.